### PR TITLE
Add video explaining how the content loader works

### PR DESCRIPTION
**Pull request by tombye**
A suggestion wrapped in a pull request. Be interested in what people think.

Adds:

image

I've pointed a few new starters to this video as a way to get their heads round the content loader and they've found it really useful.

### CODE REVIEW

**Comment by TheDoubleK** [comment]
:blush:

File: README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ schemas used by the API for validation (by running the `scripts/generate-schemas
   <img src="images/frameworks-content-triangle.png?raw=true" alt="Diagram showing how frameworks content is used by our apps">
 </p>
 
+[Kev Keenoy](https://peoplefinder.cabinetoffice.gov.uk/people/kevin-keenoy) gave a talk about how it works at PyCon UK 2016. You can watch [Kev's PyCon talk on YouTube](https://youtu.be/vdTw2Vqk4t0?t=12m44s).
```
Comment (allait) [inline README.md]: Peoplefinder is only accessible from the Cabinet Office intranet, so maybe it's better to link to Kev's GitHub account?

Comment (TheDoubleK) [inline README.md]: Or just not link me at all :)

**Comment by TheDoubleK** [comment]
:+1:

**Pull request merged**